.gitignore

```diff
@@ -0,0 +1,6 @@
+coverage.txt
+go.work
+go.sum
```

Comment:
`go.sum` should never be ignored.

According to Golang's official documentation of [Modules](https://go.dev/blog/using-go-modules):

> The go command uses the go.sum file to ensure that future downloads of these modules retrieve the same bits as the first download, to ensure the modules your project depends on do not change unexpectedly, whether for malicious, accidental, or other reasons. **Both go.mod and go.sum should be checked into version control**.

```suggestion
```